src/flowcean/transforms/flatten.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from collections.abc import Iterable
+from typing import cast, override
+
+import polars as pl
+
+from flowcean.core import Transform
+
+logger = logging.getLogger(__name__)
+
+
+class Flatten(Transform):
+    """Flatten all time series in a DataFrame to individual features.
+
+    The given DataFrame's time series are converted into individual features,
+    with each time step creating a new feature. This transform will change the
+    order of the columns in the resulting dataset.
+
+    For example the dataset
+
+    series_data              | A  | B
+    -------------------------|----|----
+    {[0, 0], [1, 1], [2, 2]} | 42 | 43
+    {[0, 3], [1, 4], [2, 5]} | 44 | 45
+
+    gets flattend into the dataset
+
+    series_data_0 | series_data_1 | series_data_2 | A  | B
+    --------------|---------------|---------------|----|----
+    0             | 1             | 2             | 42 | 43
+    3             | 4             | 5             | 42 | 43
+    """
+
+    def __init__(self, features: Iterable[str] | None = None) -> None:
```

Comment:
This function is missing documentation and also the fact that `None` leads to automatic selection.

Reply:
Thanks for noting. Should be fixed now with an explicit mention that `None` selects all possible features.